### PR TITLE
Add an htaccess file to redirect using the Accept-Language header.

### PR DESCRIPTION
**Pull request by rryan**
To test, try both http://direct.mixxx.org/manual/2.1 and https://mixxx.org/manual/2.1

### CODE REVIEW

File: htaccess [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  Header set Vary "Accept-Language"
+</IfModule>
+
+# Record the base URI in ENV::BASE.
+# https://stackoverflow.com/questions/21062290/set-rewritebase-to-the-current-folder-path-dynamically
+RewriteCond %{REQUEST_URI}::$1 ^(.*?/)(.*)::\2$
+RewriteRule ^(.*)$ - [E=BASE:%1]
+
+# First, append Accept-Language to the URL.
+RewriteRule ^$ %{REQUEST_URI}%{HTTP:Accept-Language} [L,R=301]
```
Comment (sblaisot) [inline htaccess]: doesn't L mean last, so no following rules will be used ?

File: htaccess [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# Then, hackily extract the first preferrered language by stripping everything
+# from a comma or semicolon onward.
+RewriteCond %{REQUEST_URI} [,;]
+RewriteRule ^([^,;]+).*$ %{ENV:BASE}$1 [L,R=301]
+
+# Rewrite any HTML/JS/CSS/PDF legacy links from http://mixxx.org/manual/2.1/* to
+# http://mixxx.org/manual/2.1/en/*
+RewriteCond $1 \.(html|js|css|pdf|gif|png|jpg)$
```
Comment (sblaisot) [inline htaccess]: +svg ?

Comment (rryan) [reply htaccess]: I think there were no SVGs in the manual we've published up til now.  Though I guess there are no jpgs either.

Comment (rryan) [reply htaccess]: Done

**Comment by rryan** [comment]
thanks!

**Pull request merged**